tools/cargo_bazel/Cargo.toml

```diff
@@ -30,4 +30,5 @@ textwrap = "0.14.2"
 toml = "0.5.8"
 
 [dev-dependencies]
+spectral = "0.6"
```

Comment:
I've used `spectral` a fair bit and it's been pretty solid, but I'm happy to remove it if you'd like - the benefit for me is getting nice failure messages out of assertions, rather than just that `assert!` panicking without saying what the issue was... WDYT?

Reply:
I'm fine with using it, just wanna make sure there isn't a modern equivalent that someone else would want to try and add later. 😄